### ISSUE TITLE
Code for "Learned Embedding Density"

### ISSUE THREAD

**Issue by gaozhitong** [hermannsblum/fishyscapes#18]
Hi, thanks for your great contribution!
I am interested in the "Learned Embedding Density" method proposed in the paper. Will you kindly provide the implementation code of it?

**Comment by hermannsblum** [comment]
Sorry, I don't think I have a good version to share. It was also in tensorflow 1, so not sure how useful that is...

I recommend just taking any modern implementation of normalizing flows and training it with the same parameters as we report in the paper. We also had a follow-up that had good results with gaussian mixture models instead of flows: https://arxiv.org/abs/2012.03082  
A version of this method is implemented here: https://github.com/ethz-asl/active_learning_for_segmentation/blob/main/embodied_active_learning_core/src/embodied_active_learning_core/semseg/models/uncertainty_wrapper.py

**Comment by gaozhitong** [comment]
Thanks for sharing the information! I will take a look at the paper and code.
